omero/sysadmins/index.rst

```diff
@@ -43,9 +43,53 @@ series of commands.
     unix/server-install-homebrew
     unix/server-binary-repository
     unix/server-postgresql
-    unix/install-web
     advanced-install
 
+********************
+OMERO.web Deployment
+********************
+
+OMERO.web is a python client of the OMERO platform that provides a
+web-based UI and JSON API.
+This section provides guidance on how to install and set up OMERO.web on
+any of the supported UNIX and UNIX-like platforms. Specific walkthroughs are
+provided for several systems, with detailed step-by-step instructions.
+OMERO.web can be either deployed with the OMERO.server or **separately** from 
+the OMERO.server. Deploying separately is **recommended**.
+
+
+
+However, reading through the :doc:`unix/install-web/web-deployment` first
```

Comment:
I would remove the 'However' here, it doesn't flow nicely now it isn't directly following the sentence about different walkthroughs being available.